docs/sources/installation/configuration.md

```diff
@@ -629,7 +629,7 @@ allowed_organizations =
     team_ids =
     allowed_organizations =
     ```
-
+Note: It's important to ensure that the SERVER_ROOT_URL in Grafana is set in your Azure Application Return URLs
```

Comment:
Would recommend to use the markdown quote on the beginning of your text, like `> Note...`. As you see there are other notes in this file using the markdown quote.

Also, looks like the steps guideline above doesn't mention any return url - for your note to be understandable we need to update the steps. Guessing that there are an actual field `return url` in azure portal, but cannot verify because I have no account there.

Reply:
You're correct, apologies this was rushed as I'm in the middle of the work day. I'll update this with better clarity once I am able. 

You're also correct `return url` is a field in the Azure portal.

Reply:
@Bigtalljosh any chance you want to fix the last pieces so we can merge this?

Reply:
This completely slipped the mind, sorry all, will resolve after work today.